### ISSUE TITLE
Pip can't find acceptable dependency combo for Toil 5.5.0

### ISSUE THREAD

**Issue by adamnovak** [DataBiosphere/toil#3889]
When I install `toil[all]==5.5.0` with `pip` in a fresh virtualenv, I get some error-level log messages:

```
ERROR: packaging 21.2 has requirement pyparsing<3,>=2.0.2, but you'll have pyparsing 3.0.4 which is incompatible.
ERROR: google-api-core 0.1.4 has requirement google-auth<2.0.0dev,>=0.4.0, but you'll have google-auth 2.3.3 which is incompatible.
```

We should fix our dependencies to avoid this conflict.

_Originally posted by @adamnovak in https://github.com/DataBiosphere/toil/issues/3879#issuecomment-959497542_

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/TOIL-1076)
┆friendlyId: TOIL-1076

**Comment by mr-c** [comment]
Depends on the Python version. Also, upgrade pip first?

https://github.com/common-workflow-language/cwltool/blob/0209b0b7ce66f03c8498b5a686f8d31690a2acb3/requirements.txt#L14

**Comment by adamnovak** [comment]
The `packaging` conflict no longer seems to be a problem after the `packaging` 21.3 release.

The `google-auth` conflict was caused by us pinning an old `google-storage` which was too old to share internal libraries with the version of `kubernetes` that we installed. I'm going to open it up to any 1.x `google-storage` release that's new enough, but we should eventually port over to the 2.x releases that started coming out this year, since the API looks mostly the same.
